eth/catalyst/api_test.go

```diff
@@ -443,14 +443,15 @@ func startEthService(t *testing.T, genesis *core.Genesis, blocks []*types.Block)
 	if err != nil {
 		t.Fatal("can't create eth service:", err)
 	}
-	if err := n.Start(); err != nil {
-		t.Fatal("can't start node:", err)
-	}
 	if _, err := ethservice.BlockChain().InsertChain(blocks); err != nil {
 		n.Close()
 		t.Fatal("can't import test blocks:", err)
 	}
 
+	if err := n.Start(); err != nil {
+		t.Fatal("can't start node:", err)
+	}
```

Comment:
Why is this change made? What is the race?

Reply:
No race, something I did during debugging that wasn't needed. I removed it now